Library/Formula/sourcekitten.rb

```diff
@@ -0,0 +1,16 @@
+class Sourcekitten < Formula
+  homepage "https://github.com/jpsim/SourceKitten"
+  url "https://github.com/jpsim/SourceKitten.git", :tag => "0.3.0"
+  head "https://github.com/jpsim/SourceKitten.git"
+
+  depends_on :xcode => ["6.1.1", :build]
+
+  def install
+    system "make", "prefix_install", "PREFIX=#{prefix}"
+    system "make", "clean" # remove temporary installation files in /tmp/SourceKitten.dst
```

Comment:
Since these files are written to `/tmp`, does it really matter? OS X should automatically clean up this directory if the system is low on disk space, no? This formula is based on https://github.com/Homebrew/homebrew/blob/master/Library/Formula/carthage.rb, which does the same thing.